models/enterprise_sonic/snmp/overridden_example_01.txt

```diff
@@ -1,29 +1,33 @@
 # Using overridden
-
+#
 # Before state:
-# -------------
+#---------------
 #
-#rno-dctor-1ar01c01sw02# show snmp-server community
+# show running-configuration | grep snmp
 #
-#         Community Name                     Group Name
-#-------------------------------- --------------------------------
-#private                          ro
-#public                           ro
+# snmp-server agentaddress 1.2.3.5 port 1024 interface Eth1/10
+# snmp-server user user1 group group-lab encrypted auth
+# md5 auth-password
+# U2FsdGVkX18J+L+L9OyQYWpAkGUrTgcg/6xzZoDjCbQw1ISHJ5mxmxrYZgQypEUXDeNe6rBupsc9sVDJBKxrwA==
+# priv aes-128 priv-password U2FsdGVkX1/Xs+ffZvdV9YzfyGHgIJ+zkLRPfF3/WgYIE1S4Ribvbzhu5chpHHI7ooCBpcVxYZotAXDzgetxvQ==
 #
 
-- name: Override device configuration of all SNMP communities with provided configuration
-  sonic_snmp:
-    config:
-      - community: "public123"
-        access: "ro"
-    state: overridden
+# - name: Override snmp-server configuration
+#   sonic_snmp:
+#       config:
+#           agentaddress:
+#               - ip: 1.2.3.5
+#                 port: 1024 
+#                 interface: Eth1/30
+#       state: overridden
 
-# After state:
-# ------------
+# After State:
+#---------------
 #
-#rno-dctor-1ar01c01sw02# show snmp-server community
+# show running-configuration | grep snmp
 #
-#         Community Name                     Group Name
-#-------------------------------- --------------------------------
-#public123                        ro
-#
\ No newline at end of file
+# snmp-server agentaddress 1.2.3.5 port 1024 interface Eth1/30
+# snmp-server user user1 group group-lab encrypted auth
```

Comment:
In overridden state, the "user" configuration would be deleted by the playbook shown here. Please delete all but the first line (agentaddress, etc.) from the "After State" display to show this.

Reply:
This issue can be marked "Resolved".